### DISCUSSION TITLE
Unable to install `brax` with Bazel via pip due to `orbax-checkpoint`

### DISCUSSION THREAD

**Discussion by hartikainen** [google/brax#568] [Q&A]
I'm using Bazel and installing python dependencies via pip using [`rules_python`](https://github.com/bazelbuild/rules_python). I tried upgrading `brax` from `0.11.0` to `0.12.1` but couldn't get that to work because `orbax-checkpoint`, which is a dependency of `brax`, fails to install on Bazel. This is clearly an issue within `orbax` and not `brax`, but I wanted to ask here if anyone else has seen such issue or knows any workarounds for it. I've already raised this issue (https://github.com/google/orbax/issues/1429) on their end but haven't heard back anything.

**Comment by phansel** [comment]
@hartikainen Is this still broken?

**Reply by hartikainen** [reply]
Yeah, https://github.com/google/orbax/issues/1429 and https://github.com/google/orbax/pull/1458 resolved this for us.
